test/parallel/test-fs-utimes.js

```diff
@@ -122,16 +122,30 @@ function runTest(atime, mtime, callback) {
 
 var stats = fs.statSync(__filename);
 
+
+// toUnixTimestamp
+assert.equal(fs._toUnixTimestamp('12345'), 12345);
+assert.equal(fs._toUnixTimestamp(NaN), Date.now() / 1000);
+assert.equal(fs._toUnixTimestamp(Infinity), Date.now() / 1000);
+assert.equal(fs._toUnixTimestamp(-1), Date.now() / 1000);
```

Comment:
I see. But the values should be in Epoch, so I think this test should work in only one function call.